components/reddit/reddit.app.js

```diff
@@ -0,0 +1,189 @@
+const axios = require("axios");
+const get = require("lodash.get");
```

Comment:
We follow the convention to add a newline between `require` statements and the rest of the code

Reply:
Ok noted. Added new line.